client/my-sites/themes/theme-showcase.jsx

```diff
@@ -296,6 +299,8 @@ class ThemeShowcase extends Component {
 	onFilterClick = ( tabFilter ) => {
 		const scrollPos = window.pageYOffset;
 		const isNewSearchAndFilter = config.isEnabled( 'themes/showcase-i4/search-and-filter' );
+
+		recordTracksEvent( 'calypso_themeshowcase_filter_category_click', { category: tabFilter.key } );
```

Comment:
Should this be behind feature flag `themes/showcase-i4/search-and-filter`?

Reply:
I'm fine with not putting Tracks events behind feature flags. They are cheap.